cts/cli/regression.tools.exp

```diff
@@ -2467,7 +2467,6 @@ Set 'test-primitive' option: id=test-primitive-meta_attributes-is-managed set=te
 Multiple attributes match name=is-managed
   Value: false 	(id=test-primitive-meta_attributes-is-managed)
   Value: false 	(id=test-clone-meta_attributes-is-managed)
-
```

Comment:
I wonder if it would be useful to have an out->spacer() method (newline in text, nothing in xml, maybe a line break or horizontal line in html)

Reply:
I think we've talked about that before.  I'm using out->info with an empty string to accomplish that right now.  See PCMK__OUTPUT_SPACE_IF.  The fact that it's come up twice makes me think it's a good idea.

Reply:
https://github.com/ClusterLabs/pacemaker/pull/2198